ElasticaAdminBundle/Resources/translations/messages.fr.yml

```diff
@@ -10,6 +10,9 @@ open_orchestra_elastica_admin:
         elastica_search:
             title: Recherche elastica
             description: Champs de recherche elastica
+        elastica_list:
+            title: Elastica block liste de résultat
+            description: Liste de résultat de la recherche
     form:
         elastica_list:
             search_limit: Limite de résultat
```

Comment:
I'm not sure. More in context 'limite du résultat' than 'limite des résultats', @itkg-ngilain ?